modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/bnd.bnd

```diff
@@ -1,6 +1,6 @@
 Bundle-Name: Liferay Dynamic Data Mapping API
 Bundle-SymbolicName: com.liferay.dynamic.data.mapping.api
-Bundle-Version: 8.3.2
+Bundle-Version: 8.4.0
```

Comment:
@marcosapmf, let's send to Brian, but be aware that maybe he can reject it due to the release's proximity.

Reply:
@natocesarrego, in this case, shouldn't we hold this until the release is validated?

Reply:
@marcosapmf, we can send it and see how it goes or we can talk to him on Slack, which one do your prefer?

Reply:
@natocesarrego, so I think we can send it :)